samples/msal-node-extensions/package.json

```diff
@@ -11,8 +11,8 @@
   "author": "Microsoft",
   "license": "MIT",
   "dependencies": {
-    "@azure/msal-node": "^0.1.0",
-    "express": "^4.17.1",
-    "@azure/msal-node-extensions": "0.1.0"
+    "@azure/msal-node": "^1.0.0-alpha.0",
+    "@azure/msal-node-extensions": "1.0.0-alpha.0",
+    "express": "^4.17.1"
```

Comment:
Be sure to commit the updated `package-lock.json` from this sample as well.

Reply:
Yep, will do.

Reply:
Looks like the package.json and package-lock.json were both updated in your CodeQL PR, so there were was no need to update here.